src/main/java/teammates/ui/controller/AdminActivityLogPageAction.java

```diff
@@ -156,17 +156,17 @@ private void generateStatusMessage(List<String> versionToQuery,
                 targetTimeZone = getLocalTimeZoneForUnregisteredUserRequest(courseId);
             }
         } else {
-            targetTimeZone = Const.SystemParams.ADMIN_TIME_ZONE_ID;
+            targetTimeZone = Const.SystemParams.ADMIN_TIME_ZONE;
         }
 
         String timeInAdminTimeZone =
                 TimeHelper.formatActivityLogTime(Instant.ofEpochMilli(earliestSearchTime),
```

Comment:
Can this line be on the same line as the declaration? Doesn't seem like it exceeds the line width limit.